### ISSUE TITLE
Possible to choose/compile other ssid than ESP_xxxxxx ?

### ISSUE THREAD

**Issue by zuidwijk** [jeelabs/esp-link#521]
I'm wondering, is it possible to choose another name as ssid? Of course by comping your own firmware...
I'm search through the code yet I can't find if possible, and where (if indeed it's possible).

**Comment by uzi18** [comment]
It should be possible

**Comment by zuidwijk** [comment]
Yeah ... but how/where? I've search through the whole code (did also searched with grep through the code)

**Comment by uzi18** [comment]
@zuidwijk here: 
https://github.com/jeelabs/esp-link/blob/cf329e8b84a55a05afa04df971d9abd9793c9130/Makefile#L24-L41

**Comment by zuidwijk** [comment]
Nice! Gonna try that later today ☺️

**Comment by zuidwijk** [comment]
Just tried it, unfortunately it doesn't work ....
```
# The SOFTAP configuration can be hard-coded here, the minimum parameters to set are AP_SSID && AP_PASS
# The AP SSID has to be at least 8 characters long, same for AP PASSWORD
# The AP AUTH MODE can be set to:
#  0 = AUTH_OPEN, 
#  1 = AUTH_WEP, 
#  2 = AUTH_WPA_PSK, 
#  3 = AUTH_WPA2_PSK, 
#  4 = AUTH_WPA_WPA2_PSK
# SSID hidden default 0, ( 0 | 1 ) 
# Max connections default 4, ( 1 ~ 4 )
# Beacon interval default 100, ( 100 ~ 60000ms )
#
AP_SSID ?=testname
AP_PASS ?=testname
AP_AUTH_MODE ?=0
AP_SSID_HIDDEN ?=0
AP_MAX_CONN ?=4
AP_BEACON_INTERVAL ?=100
```

AP_SSID & AP_PASS are set as they're minimum required

**Comment by uzi18** [comment]
What does it mean, ap is not shown with your name or password does not work?

**Comment by zuidwijk** [comment]
> What does it mean, ap is not shown with your name or password does not work?

It doesn't matter what I use, ... it keeps popping up with ESP_xxxxxx

**Comment by uzi18** [comment]
Did you check after complete wipe flash?

**Comment by zuidwijk** [comment]
Yeah I always wipe before flash. 
I git cloned it to another directory and then could successfully change the ssid. Yet the other settings in config.h (or .c, one off those two) now don't apply... guess have to play further with the settings.

**Comment by uzi18** [comment]
make clean command is your friend, try it in old dir

So it work for you now?

**Comment by zuidwijk** [comment]
Yeah I'll try that later (probably tomorrow)

I'm compiling via docker, so how do I make a "make clean" there?

**Comment by mamert** [comment]
Well, the initial SSID is NOT "ESP_whatever", the initial SSID is "what was last used before you flashed esp-link".
Flashing esp-link doesn't change existing wifi settings, sadly.

So, as part of the flashing procedure, you could:
1. build a small firmware (espressif or arduino) with command to start an AP **with the creds you choose**
2. flash that
3. press reset to run that code
4. flash esp-link
